code/controllers/communications.dm

```diff
@@ -142,8 +144,11 @@ var/list/radiochannels = list(
 // central command channels, i.e deathsquid & response teams
 var/list/CENT_FREQS = list(ERT_FREQ, DTH_FREQ)
 
+// Channels that show up as '#unkn'
+var/list/UNKN_FREQS = list(SYND_FREQ)
+
 // Antag channels, i.e. Syndicate
-var/list/ANTAG_FREQS = list(SYND_FREQ)
+var/list/ANTAG_FREQS = list(SYND_FREQ, SYNDTEAM_FREQ)
```

Comment:
It has to be displayed differently to the existing traitor channel, or people who have both (SIT/SST) will confuse the two. E.g: a regular traitor will ask them something on regular traitor radio, and they'll answer on their radio, never realizing its a different channel and the person they're talking to can't hear them.

It can be '#team' or something, rather than '#unkn', so its not explicitly labeled syndicate. It cannot, however, show up as the same '#unkn' the existing traitor channel uses.